src/main/java/com/dreamsportslabs/guardian/dto/request/CreateScopeRequestDto.java

```diff
@@ -0,0 +1,35 @@
+package com.dreamsportslabs.guardian.dto.request;
+
+import static com.dreamsportslabs.guardian.exception.ErrorEnum.INVALID_REQUEST;
+
+import java.util.List;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class CreateScopeRequestDto {
+  private String scope;
```

Comment:
Why do we need so many ways to create this object? 
Also its an anti pattern to have setters and Builder pattern both. 
Re think the object creation and modification aspect.